calculate.py

```diff
@@ -21,12 +26,49 @@ def evaluate(expression: str) -> str:
         expression = re.sub('(?![a-zA-Z])e(?![a-zA-Z])', 'E', expression)
         expression = expression.replace('π', 'PI')
         expression = expression.replace('√', 'sqrt')
+
+        # Convert numbers directly to rad
+        if exp_is_only_num(expression):
+            if is_rad == "true":

```

Comment:
Why is is_rad compared to a string if it's a boolean?

Reply:
Its not a boolean. Its passed to python shell as a string,  as are all parameters.

Reply:
Having looked at the code below, I now see that it's a string. In that case, the type in the method signature should be changed.